schema/home/context.js

```diff
@@ -80,13 +80,16 @@ export const moduleContext = {
       return assign({}, fair, { context_type: 'Fair' });
     });
   },
-  related_artists: ({ accessToken }) => {
-    return followedArtist(accessToken).then((follow) => {
-      return relatedArtist(accessToken).then((artist) => ({
+  related_artists: ({ params }) => {
+    return Promise.all([
+      gravity(`artist/${params.related_artist_id}`),
+      gravity(`artist/${params.followed_artist_id}`),
+    ]).then(([related_artist, follow_artist]) => {
```

Comment:
I wonder, couldn’t you use the `user/${userID}/suggested/similar/artists` endpoint instead? That will return a list of suggested artists based on the ones you follow and it includes the follows they are based on.

```
$ curl -s -H 'x-access-token: SECRET' "https://api.artsy.net/api/v1/user/USER-ID/suggested/similar/artists" | ruby -rjson -rpp -e 'pp JSON.parse(ARGF.read).map { |x| { suggestion: x["artist"]["id"], follow: x["sim_artist"]["id"] } }'

[{:suggestion=>"ken-craft", :follow=>"sandra-chevrier"},
 {:suggestion=>"kris-mortensen", :follow=>"sandra-chevrier"},
 {:suggestion=>"ian-ross", :follow=>"1010"},
 {:suggestion=>"let", :follow=>"1010"},
 {:suggestion=>"andre-saraiva", :follow=>"invader"},
 {:suggestion=>"shinji-murakami", :follow=>"invader"},
 {:suggestion=>"deborah-davidson", :follow=>"julie-bell"},
 {:suggestion=>"greg-beecham", :follow=>"julie-bell"},
 {:suggestion=>"jihye-park", :follow=>"greg-craola-simkins"},
 {:suggestion=>"kremena-lefterova", :follow=>"greg-craola-simkins"}]
```